dotnet/src/Plugins/Plugins.Web/TextSearchPlugin.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel;
+using System.Linq;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Search;
+
+namespace Microsoft.SemanticKernel.Plugins.Web;
```

Comment:
Does it make sense that it is in `Web` namespace? What about using non webpage related search services (eg. Azure AI Search?)